app/components/layouts/SecondaryNavigation.module.scss

```diff
@@ -2,6 +2,15 @@
 
 .container {
   background: $surface-4;
-  padding: $spacing-6 $spacing-16;
+  padding: $spacing-6 $spacing-8;
```

Comment:
I think you were fine with leaving it as `$spacing-16` here! (based off of the figma designs)